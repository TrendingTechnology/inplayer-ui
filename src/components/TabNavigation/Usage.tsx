import React, { useState } from 'react';
import { TabNavigation } from '.';

const Usage = () => {
  const [selectedTabIndex, setSelectedTabIndex] = useState(0);

  return (
    <TabNavigation
      tabs={[
        {
          title: 'Hello',
        },
        {
          title: 'Tabs',
        },
      ]}
      selectedTabIndex={selectedTabIndex}
      onTabClick={(index) => setSelectedTabIndex(index)}
    />
  );
};

export default Usage;
