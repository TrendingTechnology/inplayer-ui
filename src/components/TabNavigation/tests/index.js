import { generateSnapshots } from 'helpers';

import TabNavigation from '../index';

generateSnapshots(TabNavigation);
