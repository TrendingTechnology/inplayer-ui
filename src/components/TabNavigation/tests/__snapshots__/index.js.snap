// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabNavigation renders correctly 1`] = `
.c1 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-bottom: 2px solid transparent;
  color: #a0a3aa;
  cursor: pointer;
  font-weight: 300;
  line-height: 2rem;
  margin-right: 1em;
  padding: 0 4px;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: color 0.3s ease;
  transition: color 0.3s ease;
  border-color: #07AAE6;
  color: #44494e;
}

.c1:hover {
  color: #44494e;
}

.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 100%;
}

<div
  className="c0"
  style={Object {}}
>
  <a
    className="c1"
    onClick={[Function]}
  >
    Hello
  </a>
</div>
`;
